rl4co/models/nn/env_embeddings/init.py

```diff
@@ -311,6 +312,20 @@ def forward(self, td):
         return torch.cat([depot_embeddings, pick_embeddings, delivery_embeddings], -2)
 
 
+class PDPImproveInitEmbedding(nn.Module):
```

Comment:
[Minor] This does the same as the TSPInitEmbedding, so maybe we can just use that one?